packages/components/src/Button/Button.css

```diff
@@ -24,121 +13,112 @@
   min-height: calc(var(--base-unit) * 2.25);
   box-sizing: border-box;
   margin: 0;
-  border: 1px solid transparent;
+  border: var(--border-base) solid var(--button--color-variation);
   border-radius: var(--radius-base);
-  color: transparent;
+  color: var(--button--color-variation);
+  fill: var(--button--color-variation);
   -webkit-font-smoothing: antialiased;
   text-align: center;
   text-decoration: none;
   vertical-align: middle;
-  background-color: transparent;
+  background-color: var(--button--color-surface);
   cursor: pointer;
   transition: all var(--timing-base);
   appearance: none;
   align-items: center;
   justify-content: center;
 }
 
+.button * {
+  color: inherit;
+  fill: inherit;
+}
+
+.button:hover,
+a.button:hover,
 .button:focus,
-a.button:focus {
-  outline-color: var(--color-focus);
+a.button:focus,
+.button:active,
+a.button:active {
+  border-color: var(--button--color-variation--hover);
+  color: var(--button--color-variation--hover);
+  fill: var(--button--color-variation--hover);
+  background-color: var(--button--color-surface--hover);
+}
+
+.button:focus,
+a.button:focus,
+.button:active,
+a.button:active {
+  box-shadow: var(--shadow-focus);
+  outline: none;
 }
 
 /* Variation */
 
 .work {
-  border-color: var(--button--color-work);
-  color: var(--button--color-work);
-  background-color: var(--button--color-work);
-}
-
-.work:hover,
-a.work:hover,
-.work:focus,
-a.work:focus {
-  border-color: var(--button--color-work--hover);
-  background-color: var(--button--color-work--hover);
+  --button--color-variation: var(--color-interactive);
+  --button--color-variation--hover: var(--color-interactive--hover);
```

Comment:
These are redundant with the button's base CSS, but without a `.work` in here the tests blew up because the typing from css.d.ts wouldn't have a `work` string.